Source/WTF/wtf/cocoa/SoftLinking.h

```diff
@@ -244,7 +244,7 @@ static void* lib##Library() \
     } \
     _Pragma("clang diagnostic push") \
     _Pragma("clang diagnostic ignored \"-Wunused-function\"") \
-    static className *alloc##className##Instance() NS_RETURNS_RETAINED \
+    static className *alloc##className##InstanceSingleton() NS_RETURNS_RETAINED \
```

Comment:
Wait, this isn't right. What this function returns is definitely not a singleton, is it?
We're creating a new instance in each call.
We should probably just fix the static analyzer to recognize `NS_RETURNS_RETAINED` instead.